src/SimpleAzure.Storage.HybridQueue/IHybridQueueExtensions.cs

```diff
@@ -0,0 +1,20 @@
+namespace WorldDomination.SimpleAzure.Storage.HybridQueues;
+
+public static class IHybridQueueExtensions
+{
+    /// <inheritdoc cref="IHybridQueue.AddMessageAsync{T}(T, TimeSpan?, CancellationToken)" />
+    public static Task AddMessageAsync<T>(this IHybridQueue queue, T item, CancellationToken cancellationToken) =>
+        queue.AddMessageAsync(item, null, cancellationToken);
+
+    /// <inheritdoc cref="IHybridQueue.AddMessagesAsync{T}(IEnumerable{T}, TimeSpan?, int, CancellationToken)" />
+    public static Task AddMessagesAsync<T>(this IHybridQueue queue, IEnumerable<T> contents, CancellationToken cancellationToken) =>
+        queue.AddMessagesAsync(contents, null, 25, cancellationToken);
+
+    /// <inheritdoc cref="IHybridQueue.GetMessageAsync{T}(TimeSpan?, CancellationToken)" />
+    public static Task<HybridMessage<T>?> GetMessageAsync<T>(this IHybridQueue queue, CancellationToken cancellationToken) =>
+        queue.GetMessageAsync<T>(null, cancellationToken);
+
+    /// <inheritdoc cref="IHybridQueue.GetMessagesAsync{T}(int, TimeSpan?, CancellationToken)" />

```

Comment:
These inheritdoc don't make sense because you're showing the user the arguments that are not part of the method signature and that cannot be passed in.

Reply:
Yeah but it beats copying the summary all over the place 😛

I rarely doc the parameters, so I'm used to this pattern, but I guess you're right. I'm also really lazy though

Reply:
Rejecting this.

I agree with @phillip-haydon - the docs should directly reference their appropriate ones. Never be lazy IMO. This is ART! 🖼️🎨🖌️